src/route.js

```diff
@@ -6,7 +6,9 @@ module.exports = function route(API, routes={}) {
 
     // For each endpoint (create, read, update, delete...)
     return remap(resource, function(options) {
-      return API.ajax.bind(null, API.config, options)
+      let config = Object.assign({}, API.config, options)
+
+      return Object.assign(API.ajax.bind(null, config), { config })
```

Comment:
I don't want to run the risk that a configuration option includes `bind`, `call`, or `apply`, so I placed the configuration settings under `config`. Also, this is what the general object `Gangway` returns does, so I thought it would be consident.